### ISSUE TITLE
Newspaper rule does not catch functions implemented above and used within a class 

### ISSUE THREAD

**Issue by Glavin001** [Glavin001/tslint-clean-code#23]
Example:

```
function doStuff() {};

class StuffDoer {
  myMethod() {
    doStuff();
  }
}
```

Desired:

```
class StuffDoer {
  myMethod() {
    doStuff();
  }
}

function doStuff() {};
```
